packages/p2p-v2/src/pages/orders/components/OrderStatusTag/OrderStatusTag.scss

```diff
@@ -0,0 +1,23 @@
+.p2p-v2-order-status-tag {
+    border-radius: 16px;
+    padding: 0.2rem 1.6rem;
+    white-space: nowrap;
+    width: fit-content;
+
+    &--danger {
+        color: var(--brand-red-coral);
+        background-color: $color-status-error;
+    }
+    &--alert {
+        color: var(--status-warning);
+        background-color: var(--status-alert-background);
+    }
+    &--success {
+        color: var(--text-profit-success);
+        background-color: $color-status-announcement;
+    }
+    &--disabled {
+        color: var(--text-less-prominent);
+        background-color: rgba(153, 153, 153, 0.16);
```

Comment:
yes.. as of now we are using hard coded colors.. i think the deriv-com/ui team is already doing the themes so that we can use it from there.. will be having a separate card for replacing the utils and theme constants..